src/Microsoft.AspNet.Mvc.Core/Formatters/JsonOutputFormatter.cs

```diff
@@ -16,13 +17,23 @@ public class JsonOutputFormatter : OutputFormatter, IJsonOutputFormatter
         private JsonSerializerSettings _serializerSettings;
 
         public JsonOutputFormatter()
```

Comment:
I tried making this internal and a lot of test projects complained. Is InternalsVisibleTo setup? I'm not sure what I did wrong.

I did uncover a place where the default ctor is used. I'm not sure how to access the options here: https://github.com/aspnet/Mvc/blob/284c8992333d5eeede4bfd9b34974994606e5c8b/src/Microsoft.AspNet.Mvc/MvcServiceCollectionExtensions.cs#L105